phpstan.neon

```diff
@@ -15,9 +15,5 @@ parameters:
         -
             message: '#Parameter \#1 .* of method DoctrineORMModule\\Options\\Configuration.* stdClass given#'
             path: tests/Options/ConfigurationOptionsTest.php
-        -
```

Comment:
This was removed in https://github.com/doctrine/DoctrineORMModule/commit/7f112e995ab42203ceff3dd8bd8c12cba4c6939d